pennylane/workflow/construct_batch.py

```diff
@@ -118,14 +118,15 @@ def circuit():
         :class:`~.merge_rotations`.
 
         >>> qml.workflow.get_transform_program(circuit, level="user")
-        TransformProgram(cancel_inverses, merge_rotations)
+        TransformProgram(cancel_inverses, merge_rotations, _expand_metric_tensor, metric_tensor)
 
-        The ``_expand_transform_param_shift`` is the ``"gradient"`` transform.  This expands all trainable
-        operations to a state where the parameter shift transform can operate on them. For example, it will decompose
-        any parametrized templates into operators that have generators.
+        The ``_expand_transform_metric_tensor`` and ``_expand_transform_param_shift`` are the ``"gradient"`` transforms.
+        This expands all trainable operations to a state where the parameter shift transform can operate on them. For example,
+        it will decompose any parametrized templates into operators that have generators. Note that this would not include the
```

Comment:
```suggestion
        it will decompose any parametrized templates into operators that have generators. Note that ``level="gradient"`` will not include the
```